src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP-ARM64.json

```diff
@@ -50,6 +52,7 @@
       "Parameters": {
         "Scenario": "oltp_read_write_T16_TB8_REC5000",
         "DatabaseName": "sbtest",
+        "DatabaseScenario": "Default",
```

Comment:
Similarly, I would let this reference the top parameter, otherwise the top parameter wouldn't do anything